frontend/pages/upload/bmm.vue

```diff
@@ -1,64 +1,112 @@
 <script lang="ts" setup>
+import { BmmEnvironment } from "~/src/gen/api/v1/api_pb";
+import { BccSelect } from "@bcc-code/design-library-vue";
+import type { FileAndLanguage } from "~/utils/bmm";
+
 const form = ref<BMMSingleForm>({
     title: "",
+    environment: "prod",
 });
 
 const metadataIsSet = ref(false);
 
-const selectedFile = ref<File | null>(null);
+const selectedFiles = ref<FileAndLanguage[]>([]);
 
+const api = useAPI();
 const { me } = useMe();
 const config = useRuntimeConfig();
 
+const selectedEnvironment = ref(BmmEnvironment.Production);
+watch(form, (newForm) => {
+    const newEnvironment =
+        newForm.environment === "int"
+            ? BmmEnvironment.Integration
+            : BmmEnvironment.Production;
+    if (selectedEnvironment.value !== newEnvironment)
+        selectedEnvironment.value = newEnvironment;
+});
```

Comment:
a `computed` would be a better fit here
`const selectedEnvironment = computed(() => form.environment === "int" ? BmmEnvironment.Integration : BmmEnvironment.Production)`

Reply:
done